apps/notifications/src/app/templates/body.tsx

```diff
@@ -0,0 +1,140 @@
+import { __experimentalVStack as VStack, ExternalLink } from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { useState, useEffect, useMemo } from 'react';
+import { html } from '../../panel/indices-to-html';
+import { bumpStat } from '../../panel/rest-client/bump-stat';
+import { wpcom } from '../../panel/rest-client/wpcom';
+import { p, zipWithSignature } from '../../panel/templates/functions';
+import NoteActions from './actions';
+import Comment from './block-comment';
+import Post from './block-post';
+import PromptBlock from './block-prompt';
+import User from './block-user';
+import NotePreface from './preface';
+import type { Note, Block, BlockWithSignature } from '../types';
+
+const isReplyBlock = ( note: Note, block: Block ) =>
+	block.ranges && block.ranges.length > 1 && block.ranges[ 1 ].id === note.meta.ids.reply_comment;
+
+const ReplyBlock = ( { note }: { note: Note } ) => {
+	const [ replyURL, setReplyURL ] = useState< string >( '' );
+	const replyBlock = note.body.find( ( block ) => isReplyBlock( note, block ) );
+	const replyText = useMemo( () => {
+		if ( ! replyBlock ) {
+			return null;
+		}
+
+		// explicitly send className of '' here so we don't get the default of "paragraph"
+		return p( html( replyBlock ), '' );
+	}, [ replyBlock ] );
+
+	useEffect( () => {
+		if ( replyBlock ) {
+			return;
+		}
+
+		wpcom()
+			.site( note.meta.ids.site )
+			.comment( note.meta.ids.reply_comment )
+			.get( ( error: Error | null, data: { URL: string } ) => {
+				if ( ! error ) {
+					setReplyURL( data.URL );
+				}
+			} );
+	}, [ note, replyBlock ] );
+
+	if ( replyText ) {
+		return <div className="wpnc__reply">{ replyText }</div>;
+	}
+
+	if ( replyURL ) {
+		const replyMessage = createInterpolateElement(
+			note.meta.ids.comment
+				? __( 'You <a>replied</a> to this comment.' )
+				: __( 'You <a>replied<a> to this post.' ),
```

Comment:
It looks like the closing tag is missing the `/`. Should be:

`You <a>replied</a> to this post.`

Reply:
Thanks for the report! Fixing here: https://github.com/Automattic/wp-calypso/pull/105464/